jlm/llvm/ir/types.hpp

```diff
@@ -165,7 +171,7 @@ class arraytype final : public jlm::rvsdg::valuetype
 
 private:
   size_t nelements_;
-  std::shared_ptr<const jlm::rvsdg::type> type_;
+  std::shared_ptr<const jlm::rvsdg::valuetype> type_;
```

Comment:
`Type_`

Reply:
really? I thought varname are supposed to be lowercase

Reply:
@caleridas Local variables are starting with lowercase. Class attributes with capital letter and unerscore at the end.